### PR TITLE
Feature/drop 37/onboarding exercise goals

### PR DESCRIPTION
**Pull request by EEliason1**
## Describe your changes
Create onboarding exercise goals form/feature. Additional commits and file changes are present due to merges. Add to home screen items aren't relevant to the scope of this feature and are being reviewed in a different request.

## Issue ticket number and link
Drop 37
https://tripleten-apiary.atlassian.net/jira/software/projects/DROP/boards/5?selectedIssue=DROP-37

## Checklist before requesting a review

- [ x ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ x ] I have made any necessary updates to Storybook to accompany these changes
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] I have made corresponding changes to the documentation
- [ x ] My changes generate no new warnings

### CODE REVIEW

**Review by CalebHasch — CHANGES_REQUESTED** [review]
Other than the comments looks really good. Once you fix those it should be ready for final review.

File: src/routes/onboarding/set-exercise-goals/set-exercise-goals-form.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Form } from "../../../components"
+import { SetExerciseGoals } from "./set-exercise-goals-content";
+
+export const SetExerciseGoalsForm: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-lightBlue max-w-screen-md px-[10px] relative m-auto">
+      <h2 className="font-text text-section-header font-medium leader-[38.4px] text-center max-w-[358px] mx-auto">Set your exercise goals</h2>
```
Comment (RPinkha) [inline src/routes/onboarding/set-exercise-goals/set-exercise-goals-form.tsx:8]: Misspelling here. 
Line height class is leading, not leader, so it needs to be leading-[38.4px] (I would probably round it down to 38px too).

In addition, you are missing text-black to make the text #121212 like the figma requires.

File: src/routes/onboarding/set-exercise-goals/set-exercise-goals-form.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <p className="font-text text-center max-w-[358px] mx-auto mt-4 mb-8">Doing some exercise can benefit your well-being a lot!</p>
```
Comment (RPinkha) [inline src/routes/onboarding/set-exercise-goals/set-exercise-goals-form.tsx:9]: Missing "leading-[19px] text-black"

File: src/components/form/form.tsx [hunk 3]
```diff
@@ -28,5 +28,5 @@ Form.TimePicker = TimePicker;
 Form.Weekday = WeekdaySelector;
 Form.RadioGroup = RadioGroup;
 Form.SelectForm = SelectForm; 
+Form.ListTimeInput = ListTimeInput;
```
Comment (RPinkha) [inline src/components/form/form.tsx:31]: This can be reverted to how it was before

**Review by RPinkha — COMMENTED** [review]
Some things that need fixing, looks good otherwise.

**Review by jahorwitz — APPROVED** [review]

**Pull request merged**